CHANGES.md

```diff
@@ -11,7 +11,7 @@ ones in. -->
 
 --------------------------------------------------------------------------------
 
-## 2.3.0 (<span actions:bind='release-date'>Released 2024-06-18</span>)
+## 2.3.0 (<span actions:bind='release-date'>Released 2024-07-17</span>)
```

Comment:
```suggestion
## 2.3.1 (<span actions:bind='release-date'>Released 2024-07-17</span>)

Fixes an issue where the `[template variables]` section in the `rose-suite.conf` file doesn't work with Rose metadata.

## 2.3.0 (<span actions:bind='release-date'>Released 2024-06-18</span>)
```

Reply:
@wxtim, please check you're ok with this and edit / commit as desired.

Reply:
I'm pressing the button...